dolib/client.py

```diff
@@ -121,10 +121,7 @@ def request(
         data: str = None,
     ) -> t.Dict[str, t.Any]:
         response = self.request_raw(endpoint, method, params, json, data)
-        if response.status_code in [
-            requests.codes["no_content"],
-            requests.codes["accepted"],
-        ]:
+        if response.status_code in [requests.codes["no_content"]]:
```

Comment:
Maybe?

```suggestion
        if response.status_code == requests.codes["no_content"]:
```

Reply:
This is not the only code without content. API uses several. Need to add other codes)

Reply:
May be need to use http body or something else to check empty content. Because kubernetes methods return "accepted" without content but other methods use this code with content.